cylc/flow/config.py

```diff
@@ -1965,11 +1955,9 @@ def get_graph_edges(self, start_point_string, stop_point_string):
 
                     if l_id is None and r_id is None:
                         continue
-                    if l_id is not None and actual_first_point > l_id[1]:
+                    if l_id is not None and self.start_point > l_id[1]:
                         # Check that l_id is not earlier than start time.
-                        # NOTE BUG GITHUB #919
-                        # sct = start_point
-                        if r_id is None or r_id[1] < actual_first_point:
+                        if r_id is None or r_id[1] < self.start_point:
```

Comment:
Can you explain this change?  I guess `actual_first_point` is the real start cycle point, when the suite ICP is not actually on-sequence for the suite.  The same logic appears in `get_graph_raw()` but you haven't changed it there.  (It's a bit  hard to test this without any graph front-end at the moment!).

Reply:
Because the given argument `start_point` is from the pool, and the only thing I care about is that the edge source and target is not less than the ICP.. But, yes, I should check if the ICP is on sequence, so will add:
`actual_first_point = self.get_actual_first_point(self.start_point)`
This is different from `get_graph_raw` (which might need reviewed given it's not tethered to the ICP), which is:
```
start_point = get_point(start_point_string)
actual_first_point = self.get_actual_first_point(start_point)
```
(where `start_point_string` is the input arg (not ICP))